octobot_commons/profiles/profile.py

```diff
@@ -239,3 +240,31 @@ def _filter_fill_elements(
                     # save allowed keys
                     if key in allowed_keys:
                         profile_config[element][key] = value
+
+    @staticmethod
+    def get_existing_profiles_ids(profiles_path, ignore: str = None):
```

Comment:
I think you should create a `get_profiles` method that returns the list of `Profile`. Then `get_existing_profiles_ids` will only have to return the `id` of each `Profile` from `get_profiles` ? What do you think ?

Reply:
Yes, good idea !